rust-runtime/aws-smithy-http-server/src/main.rs

```diff
@@ -0,0 +1,44 @@
+// ====================
```

Comment:
This can go away or should be moved into an example if we want to keep it, right? 

Missing SPDX licence header.

Reply:
I've removed it. We can move it to an example, but it's going to require us manually writing extractors, responders, model, operation registry (essentially everything that we code-generate). So perhaps it's best we do examples as unit tests in `codegen-server`.